amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -11,7 +11,7 @@
 import urllib.parse
 import logging
 
-SEARCH_STUB_ALL_ISSUES = 'text ~ "{table_key}" order by createdDate DESC'
+SEARCH_STUB_ALL_ISSUES = 'text ~ "\\"Table Key: {table_key} [PLEASE DO NOT REMOVE]\\"" order by createdDate DESC'
```

Comment:
what's the rationale behind including the [PLEASE DO NOT REMOVE]? if there are existing tickets without that text (it was added in a later pr) they would no longer be linked

Reply:
Some would actually be. When the query was exact match on: 
`Table Key: <our-table-key>` 
an unrelated ticket was included that happened to havee:
`Table { key: <our-table-key>`

I confirmed that jira queries are not case sensitive. I did not get to the bottom of what JIRA still did an exact match on the unwanted ticket even though it had the  `{` character in the string, but I can only assume maybe they ignore that character. 

The only thing that worked in the end was to include an even more specific string.